_posts/2017-03-01-upgrade-to-babel-7-for-tool-authors.md

```diff
@@ -114,6 +114,15 @@ const helperPaths = path.unshiftContainer("body", helpers);
 
 ## AST changes
 
+### JSX* and TS* nodes renamed
```

Comment:
Maybe `node builders` instead of `nodes`?

Reply:
Yes!